docs/source/upgrade-guide/index.md

```diff
@@ -1237,14 +1239,14 @@ export default function applyConfig(config) {
 }
 ```
 
-````{warning}
+```{warning}
 The add-ons you might be using might need to migrate to use the new
 configuration registry too. Make sure all of them are already migrated to Volto 12.
 
 ```{tip}
 Although this might be daunting, the migration is quite straightforward, and the refactoring
 of the required code can be undergone through a series of "search and replace" in your IDE of choice.
-````
+```
```

Comment:
The tip should be nested within the warning.

`````
````{warning}
The add-ons you might be using might need to migrate to use the new
configuration registry too. Make sure all of them are already migrated to Volto 12.

```{tip}
Although this might be daunting, the migration is quite straightforward, and the refactoring
of the required code can be undergone through a series of "search and replace" in your IDE of choice.
```
````
`````